### DISCUSSION TITLE
Ability to have News feed use custom external link anchor tag

### DISCUSSION THREAD

**Discussion by TryTryAgain** [grafana/grafana#38260] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

The ability to use a custom link tag for a news feed external link to article.

**Why is this needed**:

Currently, it seems the News feed only uses what's wrapped in `<link></link>` tags. As seen in this feed: https://status.aws.amazon.com/rss/all.rss the link tag is unfortunately generic and we actually want to use `<guid></guid>` for example.

I'm not sure how common/uncommon that is. I'm guessing it would be helpful in other cases regardless.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by TryTryAgain** [reply]
Too bad, thanks.
